### PR TITLE
Add meta example for Sparse Euclidian Distance #3000

### PR DESCRIPTION
**Pull request by Hephaestus12**
Changes to be committed:
	new file:   examples/meta/src/distance/**sparseeuclidean.sg**
	deleted:    examples/undocumented/python/**distance_sparseeuclidean.py**

### CODE REVIEW

File: examples/meta/src/distance/sparseeuclidean.sg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+CSVFile f_feats_a("../../data/fm_train_sparsereal.dat")
```
Comment (vigsterkr) [inline examples/meta/src/distance/sparseeuclidean.sg:1]: wrong file name.... should be `sg.in`

**Review by vigsterkr — CHANGES_REQUESTED** [review]

File: examples/meta/src/distance/sparseeuclidean.sg (hunk 1 above)
Comment (gf712) [reply examples/meta/src/distance/sparseeuclidean.sg:1]: also use `SHOGUN_DATA` instead of relative path

Comment (Hephaestus12) [reply examples/meta/src/distance/sparseeuclidean.sg:1]: I don't understand.
This was the filename in the existing python example as well.
Also, I don't see any `sg.in` file in the `data` directory.

Comment (vigsterkr) [reply examples/meta/src/distance/sparseeuclidean.sg:1]: it should be `examples/meta/src/distance/sparseeuclidean.sg.in` you'll see that all other examples have that extension....

and plz check the other examples for how the data file path is being defined.

Comment (vigsterkr) [reply examples/meta/src/distance/sparseeuclidean.sg:1]: and as a general rule it'd be great if you could test (compile and run) any patch *before* you create a pull request

Comment (karlnapf) [reply examples/meta/src/distance/sparseeuclidean.sg:1]: @Hephaestus12 just add make it more clear, @vigsterkr was not talking about the filename in the csv file, but about the filename of the meta example listing itself

**Pull request closed without merging**